site/content/3.12/release-notes/version-3.12/whats-new-in-3-12.md

```diff
@@ -222,5 +222,59 @@ of outgrowing the maximum number of file descriptors the ArangoDB process
 can open. Thus, these options should only be enabled on deployments with a
 limited number of collections/shards/indexes.
 
+## Client tools
+
+### arangodump
+
+#### Improved dump performance
+
+ArangoDB 3.12 includes extended parallelization capabilities to work not only
+at the collection level, but also at the shard level. In combination with the
+new optimized format, database dumps are now created and restored quickly and
+occupy minimal disk space. This major performance boost makes dumps five times
+faster and restores three times faster, which is extremely useful when dealing
+with large shards.
```

Comment:
Is the `--parallel-dump` option that controls whether the new dumping implementation is used not mentioned on purpose? The default appears to be false (option disabled):
https://github.com/arangodb/arangodb/blob/4f154796969214eca5c25e143afb87fb9a16060d/client-tools/Dump/DumpFeature.h#L86

Reply:
Correct, it was indeed still `false` by default.
I haven't had time to make it the default. But now there is upstream PR https://github.com/arangodb/arangodb/pull/20022, which should make it the default.